Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -1890,9 +1915,6 @@ extension TestStore where ScopedState: Equatable {
         return
       }
     }
-
-    guard !Task.isCancelled
-    else { return }
```

Comment:
I think this doesn't do anything since both branches returned. Let me know if you want it back.